automated-comments/go/hamming/declare_variable_when_needed_not_start_of_function.md

```diff
@@ -0,0 +1,4 @@
+Variably %{returngVar} is earlier than necessary.
```

Comment:
I'm not sure what this means. Is it supposed to be something like this?

```
Variable %{returngVar} is defined earlier than necessary.
```

Reply:
Yes, thx!